Content.Client/Content.Client.csproj

```diff
@@ -23,6 +23,11 @@
     <ProjectReference Include="..\RobustToolbox\Robust.Client\Robust.Client.csproj" />
     <ProjectReference Include="..\Content.Shared\Content.Shared.csproj" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Do XAML's typically need an explicit dependency to compile correctly?

Reply:
That code was auto-generated, so I have no clue